src/plugins/storage.js

```diff
@@ -17,12 +17,6 @@ class Storage {
 	}
 
 	dereference(url) {
-		// If maxHistory is 0, image would be dereferenced before client had a chance to retrieve it,
-		// so for now, just don't implement dereferencing for this edge case.
-		if (helper.maxHistory === 0) {
```

Comment:
Sorry, what I meant was: why is this needed in one case and not every time you call `dereference`?